### PR TITLE
Rename replay debugger

### PR DESCRIPTION
**Pull request by DatGuyJonathan**
### What does this PR do?
Changes the extension name. I'm not changing the folder name yet; I think that can be done when the adapter is separated from the extension.

### What issues does this PR fix or reference?
@W-5014405@

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -356,7 +356,7 @@
       "request": "launch",
       "cwd": "${workspaceRoot}",
       "program":
-        "${workspaceRoot}/packages/salesforcedx-vscode-replay-debugger/out/src/adapter/apexReplayDebug.js",
+        "${workspaceRoot}/packages/salesforcedx-vscode-apex-replay-debugger/out/src/adapter/apexReplayDebug.js",
```
Comment (vazexqi) [inline .vscode/launch.json]: @DatGuyJonathan - I'm confused. You said that you're not changing the name of the folder yet but I see that you change the name here in launch.json so the folder name needs to change as well or else we won't be able to launch anymore?

Comment (DatGuyJonathan) [reply .vscode/launch.json]: Oops, reverting.

**Comment by vazexqi** [comment]
@DatGuyJonathan - I have a question after thinking about this a bit more.

Could you elaborate more on why shouldn't we just rename the folder as well to salesforcedx-vscode-apex-replay-debugger? Even if we decide to split out the adapter, won't the name just be salesforcedx-apex-replay-debugger (without the vscode in it just like how we treat the interactive debugger adapter folder)?

Asking because right now it feels a bit weird with the folder name being out of sync with the package.json file.

**Comment by vazexqi** [comment]
Got it. Thanks for elaborating. Let's do it after those changes are in then.

**Pull request merged**